gcm_filters/kernels.py

```diff
@@ -141,9 +158,122 @@ def __call__(self, field: ArrayType):
         return out
 
 
-ALL_KERNELS[
-    GridType.IRREGULAR_CARTESIAN_WITH_LAND
-] = IrregularCartesianLaplacianWithLandMask
+ALL_KERNELS[GridType.IRREGULAR_WITH_LAND] = IrregularLaplacianWithLandMask
+
+
+@dataclass
+class TripolarRegularLaplacianTpoint(BaseLaplacian):
+    """̵Laplacian for fields defined at T-points on POP tripolar grid geometry with land mask, but assuming that dx = dy = 1
+
+    Attributes
+    ----------
+    wet_mask: Mask array, 1 for ocean, 0 for land
+    """
+
+    wet_mask: ArrayType
+
+    def __post_init__(self):
+        np = get_array_module(self.wet_mask)
+
+        # check that southernmost row of wet mask has only zeros
+        if self.wet_mask[..., 0, :].any():
+            raise AssertionError("Wet mask requires zeros in southernmost row")
```

Comment:
@rabernat, related to your comment on checks [here](https://github.com/ocean-eddy-cpt/gcm-filters/pull/28#discussion_r594690734), I'm assuming you would suggest to remove this check (and associated test) as well. I added this check because this is a global Laplacian (on a tripole grid), but the southern boundary must be land in order to disable the periodic boundary condition in y-direction. Otherwise, the filtered values in the very south will be contaminated by stuff in the Arctic. 

Again, an alternative is to document the requirement of wet mask zeros in the southernmost row carefully, and I am happy to remove this check.

Reply:
Good point.

I will think about this and get back to you. Perhaps fixing the problem at the root (only call `__post_init__` once) would be the best path forward.

Reply:
I did some profiling and decided that this is not actually a big performance hit. So we can leave things as are for now.